### ISSUE TITLE
Cannot SIgn in on Android F-Droid release

### ISSUE THREAD

**Issue by tokenwizard** [tailscale/tailscale#2580]
I created an account via my Github identity. I am able to login fine on my desktop web browser. However on my Android device I cannot get signed in fully. 

I'm running Tailscale v1.11.106-1bb6abc604c-g2c1f35d5607 downloaded from F-Droid

I launch the App and get the Sign In button. When I click that, it opens an instance of my mobile browser (Firefox Nightly) and give me the option to log in with Google, Microsoft, or Github. I click the button for GitHub and it asks me to authorize the Tailscale app (which I already did via the Desktop). I click the green Authorize button and it takes me to a screen that says Authorization Successful. You may close this window. When closing the window I am back at the Tailscale app and it still just shows the Sign In button.

It seems that I did add the device, because on the Desktop, when I look at my settings, I see my Android Device listed. But in the Tailscale Android App, it seems I am not signed in. 

Screenshot sequence:
![screen1](https://user-images.githubusercontent.com/24832561/128195952-c06f23af-90cc-4965-bc0a-35ac78300b96.png)
![screen2](https://user-images.githubusercontent.com/24832561/128195965-830d561a-b8fb-4418-8f16-eaa7f2a0ba39.png)
![screen3](https://user-images.githubusercontent.com/24832561/128195982-210678dc-97ee-4a58-bc8c-2601128540d6.png)
![screen4](https://user-images.githubusercontent.com/24832561/128195999-c9c4661e-8773-4f87-a231-223dca3a22f5.png)
![screen5](https://user-images.githubusercontent.com/24832561/128196023-41276558-e98d-48bd-9467-474957b4cf9e.png)
![screen6](https://user-images.githubusercontent.com/24832561/128196036-efd2dd31-74cd-424e-aa68-f9ae6974c6f9.png)

View of my dashboard from the Desktop/Web:
![image](https://user-images.githubusercontent.com/24832561/128196409-4176b8ca-08c6-4aac-9e31-fff8bb2cacbe.png)

I notice it says Update Available, but I am running the latest version from F-Droid. Is that Repo not up to date?

**Comment by bradfitz** [comment]
I filed https://github.com/tailscale/tailscale/issues/2581 for the "Update Available" bug.

**Comment by DentonGentry** [comment]
A user on https://github.com/tailscale/tailscale/issues/2478 who was experiencing the same login issue with 1.11.106 reported that it was resolved by 1.11.151, which is available for download on http://f-droid.org/en/packages/com.tailscale.ipn/

**Comment by tokenwizard** [comment]
Ok, I confirmed with the newer version of the app it does allow me to complete the login. Thanks for the assistance.

I reinstalled F-Droid to ensure I had the latest version of that app, then refreshed to Update Repositories. After doing so, it offered me the 1.11.151 version within F-Droid. 

Your related bug [#2581](https://github.com/tailscale/tailscale/issues/2581) is still relevant to the latest version from F-Droid. On the Dashboard it still shows that device as having an update available.
